### PR TITLE
NUbook Update January 2021

### PR DESCRIPTION
**Pull request by ysims**
Updates areas of NUbook

- Members list
- Publications (https://github.com/NUbots/NUbook/issues/70)
- Recruitment information
- Getting started page
- README link fix
- Page numbering
- Heading levels
- Minor wording changes

### CODE REVIEW

File: src/book/01-team/01-introduction/03-current-team.mdx [hunk 1]
```diff
@@ -14,75 +14,84 @@ slug: /team/current-members
...
 
+- **Matt Amos**: Computer Science and Computer Engineering Honours graduate. Specialises in synthetic dataset generation and robotic movement.
+
 - **Brendan Annable**: Bachelor of Software Engineering Graduate. Ex-Team Leader. Lead Developer of NUsight.
 
-- **Dr. Trent Houliston**: Holds a Doctorate of Philosophy in Software Engineering. Ex-Team Leader. NUbots mentor. Currently working at 4Tel. Specialises in Software Architecture, Machine Learning and Computer Vision.
+- **Alex Biddulph**: Bachelor of Computer Science, Bachelor of Computer Engineering with Honours. Currently studying for a Doctorate of Philosophy in Computer Engineering. Ex-Team Leader. Specialises in computer vision.
 
-- **Matt Amos**: Computer Science and Computer Engineering Honours graduate. Specialises in synthetic dataset generation and robotic movement.
+- **Dr. Trent Houliston**: Holds a Doctorate of Philosophy in Software Engineering. Ex-Team Leader. NUbots mentor. Currently working at 4Tel. Specialises in Software Architecture, Machine Learning and Computer Vision.
```
Comment (KipHamiltons) [inline src/book/01-team/01-introduction/03-current-team.mdx:94]: Alex and Trent specialise? I thought they knew everything ;P

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: Solutions to common problems you might encounter at NUbots.
 slug: /guides/general/troubleshooting
 ---
 
-# General
```
Comment (KipHamiltons) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:9]: I thought skipping header levels was naughty?

**Review by KipHamiltons — APPROVED** [review]
Make sure the renamed files don't break any links (a tool for this would indeed be lit).

One question about headers in `troubleshooting`. Otherwise, LGTM

File: src/book/03-guides/04-general/02-troubleshooting.mdx (hunk 2 above)
Comment (ysims) [reply src/book/03-guides/04-general/02-troubleshooting.mdx:9]: Not the first level. The first level is whatever the title of the page is, eg Troubleshooting. Therefore we should start all headers from level 2.

File: src/book/01-team/01-introduction/03-current-team.mdx (hunk 1 above)
Comment (ysims) [reply src/book/01-team/01-introduction/03-current-team.mdx:94]: :laughing: you're not wrong

**Comment by ysims** [comment]
The renamed files wont break any links - it's changing the slug that will.

**Review by KipHamiltons — APPROVED** [review]

**Pull request merged**